core/src/stories/Migration/Migration.stories.mdx

```diff
@@ -20,10 +18,21 @@ Previously we have not had a set naming convention for our events. This has led
 components where some events has had a `custom` prefix, and others have had other ways of indicating whether
 or not it is a custom event. We have now settled on a naming convention for all events.
 Both **external**, events we expect you as a user to interact with, and **internal**, events that are used within
-our components to update state/react to changes.
+our components to update state/react to changes. 
 
-In short:
+## Prefix change
+Furthermore we decided to rename the previously used 'sdds'-prefix to 'tds'. 
+This occurred as a result of a deliberate choice to facilitate the transition to the new Tegel package and prevent conflicts with previously used components.
 
+In short, before the prefix change:
+
+- Web component names are named with 'sdds-' + component name. ('sdds-badge')
+- External events are named: 'sdds' + native event name. ('sddsClick', 'sddsChange')
+- Internal events are named: 'internalsdds' + event name. ('internalsddsTablePropsChange', 'internalsddsCheckboxChange')
+
+Now, after prefix change:
+
+- Web component names are named with 'tds-' + component name ('tds-badge')
 - External events are named: 'tds' + native event name. ('tdsClick', 'tdsChange')
 - Internal events are named: 'internaltds' + event name. ('internaltdsTablePropsChange', 'internaltdsCheckboxChange')
 
```

Comment:
I know it's not part of this PR. But do you want to update the link below that has a TODOLINKTOCODE href.

Reply:
Sure I can fix it! Although, think I will close this PR and open a new one. Too many merge clonflicts. @mJarsater

Reply:
Sure thing!

Reply:
Btw, where do we wanna link to? Here https://github.com/scania-digital-design-system/tegel/blob/main/.github/CODE_STYLE.md

Reply:
Yeah that makes sense. May the sentence should be tweked a bit.  Read more about or naming conventions in our code convetions or something?

Reply:
Sounds good! Ill figure something out.